src/ui/component/fileActions/view.jsx

```diff
@@ -15,15 +16,27 @@ type Props = {
   openModal: (id: string, { uri: string }) => void,
   claimIsMine: boolean,
   fileInfo: FileInfo,
+  showFullscreen: boolean,
 };
 
 class FileActions extends React.PureComponent<Props> {
+  maximizeViewer = () => {
+    // Get viewer container
+    const viewer = document.getElementsByClassName('content__embedded')[0];
```

Comment:
This should use a ref. We will need probably need to pass it to this component with `forwardRef`. Or we could use `useRef`, but that would involve turning the component that uses `content_embedded` to be a function component so it can use context/hooks.